README.md

```diff
@@ -155,6 +163,19 @@ To unload the test kext:
 the signed one installed to `/Library/Extensions/`:
 `sudo kextload /Library/Extensions/intelhaxm.kext`
 
+### Viewing kext logs
+On macOS, HAXM debug output goes to the system log database, and can be
+retrieved at almost any time.
+
+* On OS X 10.10, HAXM log messages are written immediately to
+`/var/log/system.log`. They can be viewed conveniently using _Console.app_ or
+the `syslog` command.
```

Comment:
dmesg cmd could be used to dump mac kernel message, do we need to mention it?

Reply:
Thanks! `dmesg` is pretty useful despite a few limitations.